src/main/java/org/circle8/service/PuntoReciclajeService.java

```diff
@@ -31,4 +33,16 @@ public List<PuntoReciclajeDto> list(PuntoReciclajeFilter filter) throws ServiceE
 			throw new ServiceError("Ha ocurrido un error al obtener el listado de puntos de reciclaje", e);
 		}
 	}
+
+	/**
+	 * Obtiene un punto de reciclaje por medio de su id
+	 */
+	public PuntoReciclajeDto get(Long id, Long recicladorId) throws ServiceError, NotFoundException {
+		try {
+			return PuntoReciclajeDto.from(this.dao.get(id, recicladorId).
+				orElseThrow(() -> new NotFoundException("No existe el punto de reciclaje")));
```

Comment:
Te paso una version mas bonita (porque lo digo yo :hear_no_evil:)
```suggestion
			return this.dao.get(id, recicladorId)
				.map(PuntoReciclajeDto::from)
				.orElseThrow(() -> new NotFoundException("No existe el punto de reciclaje"));
```